src/module/module.jl

```diff
@@ -457,11 +457,21 @@ function Module(R::PolyRing{T}, vecs::svector{spoly{T}}...) where T <: Nemo.Ring
    return smodule{S}(R, vecs...)
 end
 
+function Module(R::PluralRing, vecs::svector{spluralg{T}}...) where T
+   S = elem_type(R)
+   return smodule{S}(R, vecs...)
+end
+
 function Module(R::PolyRing{T}, id::libSingular.ideal_ptr) where T <: Nemo.RingElem
    S = elem_type(R)
    return smodule{S}(R, id)
 end
 
+function Module(R::PluralRing, id::libSingular.ideal_ptr) 
```

Comment:
This is exact same as the previous method, just with the `T` dropped, isn't it? So then you can remove the previous method? Or keep it, change the `Nemo.RingElem` to `NCRingElem` (I think the `Nemo.` prefix can be dropped these days, too), and use that?